provider-directory/src/main/java/gov/va/api/health/providerdirectory/service/VlerResponse.java

```diff
@@ -0,0 +1,23 @@
+package gov.va.api.health.providerdirectory.service;
+
+import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+public interface VlerResponse {
```

Comment:
If the `error` body on VLER messages isn't applicable, we can remove this interface.

Reply:
Error body removed.